nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-documentation/src/main/java/org/apache/nifi/documentation/DocGenerator.java

```diff
@@ -57,6 +57,9 @@
      * @param extensionMapping extension mapping
      */
     public static void generate(final NiFiProperties properties, final ExtensionMapping extensionMapping) {
+
```

Comment:
these extra lines should be removed

Reply:
yes, indeed, this somehow slipped in: I'll fix in my fork and create a new pull request later on.